easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-11.3.0.eb

```
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'UDA'
+version = '2.6.0'
+
+homepage = 'https://github.com/ukaea/UDA'
+description = """Universal Data Access"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ukaea/UDA/archive/refs/tags']
+sources = ['%(version)s.tar.gz']
+checksums = ['9fd8b917a4f6bc0eaadb04cc6b04fb98cebcdee7f5a31ec7bc2604839ca2e395']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('libMemcached', '1.0.18'),
+    ('libtirpc', '1.3.2'),
+]
+
+dependencies = [
+    ('libxml2', '2.9.13'),
+    ('Boost', '1.79.0', '-no_mpi'),
```

we already build `Boost` with the tagged layout, no need to introduce a `-no_mpi` variant
```suggestion
    ('Boost', '1.79.0'),
```